Makefile

```diff
@@ -129,6 +129,23 @@ test-containerized: run-etcd run-k8s-apiserver build-containerized dist/host-loc
 			ginkgo'
 	make stop-etcd
 
+.PHONY: k8s-install/scripts/install_cni.test
```

Comment:
@heschlie just noticed this - I don't think should be marked as .PHONY

Also, we need to add this binary to `make clean`

Reply:
Ah you're right, it spits out a binary, standby.

Reply:
OK, fixed and kept squashed.